index.d.ts

```diff
@@ -527,6 +527,8 @@ declare namespace Moleculer {
 		needAck: boolean | null;
 		ackID: string | null;
 
+		locals: any | null;
```

Comment:
Please be more strict. `ctx.local` always an object and can't be null. So use `local: GenericObject`

Reply:
Thank you.
Changed to
```ts
locals: GenericObject;
```